app/models.py

```
@@ -132,28 +403,31 @@ class LeaveBotRequest(BaseModel):
 class PersonaImageResponse(BaseModel):
     """Response model for generated persona images."""
+
     name: str = Field(..., description="Name of the persona")
     image_url: str = Field(..., description="URL of the generated image")
     generated_at: datetime = Field(..., description="Timestamp of generation")
 
     class Config:
-        json_encoders = {
-            datetime: lambda v: v.isoformat()
-        }
+        json_encoders = {datetime: lambda v: v.isoformat()}
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)